client/blocks/inline-help/style.scss

```diff
@@ -383,14 +385,21 @@
 		}
 
 		.gridicon {
-			align-self: baseline;
+			align-self: center;
```

Comment:
Changing this puts the icon in the middle when the item is multiline.

<img src="https://user-images.githubusercontent.com/77539/86963883-9c6f6a00-c13b-11ea-8872-15e097afab5e.png" width="400px" />

If I'm not wrong, we want to align it at the top, do we

Reply:
I added it back in: https://github.com/Automattic/wp-calypso/pull/44004/commits/9bbd276842eaaff4d2a4b2e89a2a42a23cde0ad5, and made sure the icon doesn't shrink in 2-line results.

Reply:
![image](https://user-images.githubusercontent.com/77539/86966218-2967f280-c13f-11ea-9c11-05c6dd4a0bca.png)